### ISSUE TITLE
Can't make light icon with transparent background

### ISSUE THREAD

**Issue by ChrisNelsonPE** [Templarian/MaterialDesign#2144]
If I go to https://materialdesignicons.com/ and click on an icon, a dialog pops up with a preview, the name, description, tags, aliases and buttons for Advanced Export, etc.

If I click Advanced Export, I see a preview and a couple of color palettes.  If in the Foreground palette I click the black in the lower right corner, all of the background turns transparent and I can download a black icon to be used on any light background.

If I click on the white box above that, the background reverts to black and I downloading gives me a white icon on a black background which looks odd on a navy or red or whatever background.

If I click on the black box in the lower right corner of background then drag the slider below to the right, the background *outside* the icon goes transparent but not the background with the icon dimensions.

Is this a bug or am I missing something?

**Comment by Templarian** [comment]
The checkboard transparent pattern changes colors based on various factors.

- If background opacity > 0 then use background color to calculate white/black checkerboard
- If background opacity = 0 then use foreground color to calculate white/black checkboard

It's all about showing contrast that's easier to see the icon on. Unless you're talking about something else, currently coding the new website, but the legacy site should be relatively bug free (just slow).

**Comment by ChrisNelsonPE** [comment]
I'm definitely not talking about just how the transparency is presented but how it works once the icon is downloaded.

For anyone else who ends up here, I'll mention that all I had to do was click down the Icon button on the simple export and pick black or white then the rest of the downloaded icon is transparent.  I didn't need the Advanced Export at all.  I imagine the problem I described persists if I wanted to pick another color but this was enough for me.
